app/grandchallenge/components/models.py

```diff
@@ -1143,6 +1150,14 @@ def _clean_store_in_database(self):
                 f"Interface {self.kind} objects cannot be stored in the database"
             )
 
+    def _clean_example_value(self):
```

Comment:
Example values should only be set when the kind is `is_json_kind`. Tests should be added for that.

Reply:
Do you mean it should raise a validation error if the example was provided with non json kind interfaces?
If so, then no need for extra tests for it, correct?

Reply:
Yes. Why do you think that it wouldn't need a test?

Reply:
ah, I misintepreted it, my bad. Of course, it needs a test that fails on interfaces with non-json kinds when having an example value (and tests for correct cases).

Reply:
Done